sample/build.gradle

```diff
@@ -70,6 +70,10 @@ if (project.hasProperty("useKsp")) {
 }
 
 
+ksp {
+    arg("MultipreviewTypes", "com.airbnb.android.showkasesample.CustomPreview")
+}
```

Comment:
I think I can remove this now. Might need it for KAPT, but works otherwise with KSP :)